fm.reaper.Reaper.yml

```diff
@@ -0,0 +1,70 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+tags:
+ - proprietary
+command: /app/REAPER/reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --device=all
+  - --filesystem=home
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --system-talk-name=org.freedesktop.Flatpak
+  - --env=ALSA_CONFIG_PATH=
+  - --env=LV2_PATH=/app/extensions/Plugins/lv2
+  - --env=DSSI_PATH=/app/extensions/Plugins/dssi
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --env=VST_PATH=/app/extensions/Plugins/lxvst
+  - --env=VST3_PATH=$HOME/.vst3:/app/extensions/Plugins/vst3
+
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /cache
+  - /include
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '21.08'
+    add-ld-path: lib
+    merge-dirs: ladspa;dssi;lv2;lxvst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '21.08'
+    add-ld-path: "."
+    autodelete: false
+
+cleanup-commands:
+- mkdir -p /app/lib/ffmpeg
+  
+
+modules:
+  - name: reaper
+    buildsystem: simple
+    build-commands:
+      - sh install-reaper.sh --quiet --install /app
+      - install -Dm644 /app/REAPER/Resources/main.png /app/share/icons/hicolor/256x256/apps/fm.reaper.Reaper.png 
+      - install -Dm644 fm.reaper.Reaper.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+      - desktop-file-validate /app/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: archive
+        url: https://www.reaper.fm/files/6.x/reaper658_linux_x86_64.tar.xz
```

Comment:
since this is x86_64 only, you need to tell Flathub to not build anything else.

See https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64